cpu/atmega_common/posix_unistd.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
```

Comment:
well, you are right, but there are several files that do it anyway. Which doesn't make it better or the right thing to do, though. Maybe RIOT needs some proper guidelines on when and how to use doxygen tags?